proposal/BoWeii/README.rst

```diff
@@ -0,0 +1,115 @@
+Job shop scheduling
+===================
+
+Basic information (including the GitHub repository)
+===================
+
+Job-shop scheduling or the job-shop problem (JSP) is an optimization
+problem in computer science and operations research.
+
+I would try to use genetic-algorithm solve the JSP.
+
+**GitHub repository**: https://github.com/BoWeii/job-shop-scheduling 
+
+Problem to solve
+===================
+In a general job scheduling problem, we are given n jobs **J1, J2, ...,
+Jn** of varying processing times,each job consists of a set of
+operations **O1, O2, ..., On** which need to be processed in a specific
+order (known as precedence constraints). Each operation has a specific
+machine that it needs to be processed on and only one operation in a job
+can be processed at a given time. We need to schedule the jobs on **m**
+machines with varying processing power, while trying to minimize the
+makespan. |image0|
+
+Perspective users
+===================
+
+Some automated processing factories or people with similar needs will
+need this algorithm
+
+System architecture
+===================
+-  **Requure parameters and datas**
+
+   1. population size
+   2. crossover rate 
+   3. mutation rate 
+   4. processing table for Jobs/Operations
+   5. machine sequence table for Jobs/Operations
+   
+-  **Python:**
+
+   1. initialize the informantion and parameters 
+   2. output the time consume
+   3. render the result as Gantt Chart
+
+
+-  **C++:**  
+
+   Responsibility for all of Genetic-Algorithm working
```

Comment:
It is OK to be a little bit vague for the numeric method at the moment, but it is still good to provide something about it.  I can be wrong, but lack of the basic descriptions looks like a sign that you do not know well about what genetic algorithm does.  Figuring out some basic information early will help you a lot.

Reply:
I’m sorry about that I just leave the concise text, I fix this part already.

Reply:
The newly added list of operations shows that you have a clear compartment between the high-level API (in Python) and low-level implementation (in C++).  It looks good.

Also keep in mind that during your implementation the compartment may move.  It the change of the (internal) module boundary makes sense, you can just change it.

BTW, it is not needed to apologize for lack of information in a draft.  Let's focus on discussions that help your proposal.